selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -317,7 +319,50 @@ void DevicePanel::updateCalibDescription() {
       qInfo() << "invalid CalibrationParams";
     }
   }
-  desc += tr(" Resetting calibration will restart openpilot if the car is powered on.");
+
+  int lag_perc = 0;
+  std::string lag_bytes = params.get("LiveDelay");
+  if (!lag_bytes.empty()) {
+    try {
+      AlignedBuffer aligned_buf;
+      capnp::FlatArrayMessageReader cmsg(aligned_buf.align(lag_bytes.data(), lag_bytes.size()));
+      auto lag = cmsg.getRoot<cereal::Event>().getLiveDelay();
+      lag_perc = lag.getCalPerc();
+    } catch (kj::Exception) {
+      qInfo() << "invalid LiveDelay";
+    }
+  }
+
+  if (lag_perc < 100) {
+    desc += tr("\n\n- Learning the vehicle's steering lag: %1% complete.").arg(lag_perc);
+  } else {
+    desc += tr("\n\n- Vehicle's steering lag learning complete.");
+  }
+
+  std::string torque_bytes = params.get("LiveTorqueParameters");
+  if (!torque_bytes.empty()) {
+    try {
+      AlignedBuffer aligned_buf;
+      capnp::FlatArrayMessageReader cmsg(aligned_buf.align(torque_bytes.data(), torque_bytes.size()));
+      auto torque = cmsg.getRoot<cereal::Event>().getLiveTorqueParameters();
+
+      // don't add for non-torque cars
+      if (torque.getUseParams()) {
+        int torque_perc = 100;//torque.getCalPerc();
+        if (torque_perc < 100) {
+          desc += tr("\n- Learning the vehicle's steering response: %1% complete.").arg(torque_perc);
+        } else {
+          desc += tr("\n- Vehicle's steering response learning complete.");
```

Comment:
complete is not correct since we continually update the estimates. going to not show if complete for now